ynr/apps/people/models.py

```diff
@@ -373,6 +373,18 @@ def get_edit_url(self, request=None):
             return path
         return request.build_absolute_uri(path)
 
+    def queued_image(self):
+        queued_image = self.queuedimage_set.filter(decision="undecided").first()
+        if queued_image:
+            return queued_image.image
+
+    def get_absolute_queued_image_url(self):
+        queued_image = self.queuedimage_set.filter(decision="undecided").first()
+        if queued_image:
```

Comment:
You could use your above method here instead of repeating the query:
```
if not self.queued_image:
    return None
return self.queued_image.get_absolute_url()
```
If you do this, it would be worth making `queued_image` a cached_property to save multiple DB queries